### PR TITLE
feature: Cross compile to Scala 2.11.12 Play version 2.4.3

### PR DESCRIPTION
**Pull request by machadoit**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -30,12 +30,14 @@ val stashScalaClient = project
   )
   .settings(unmanagedSourceDirectories.in(Compile) += {
     Dependencies.playJson(scalaVersion.value).head.revision match {
-      case Dependencies.playJson25 => baseDirectory.in(Compile).value / "src" / "main" / s"play_json_2.5"
+      case (Dependencies.playJson24 | Dependencies.playJson25) =>
+        baseDirectory.in(Compile).value / "src" / "main" / s"play_json_2.5-"
```
Comment (lolgab) [inline build.sbt]: You can remove the `s`. The same in the next line
```suggestion
        baseDirectory.in(Compile).value / "src" / "main" / "play_json_2.5-"
```

Comment (machadoit) [reply build.sbt]: 👍

File: project/Dependencies.scala [hunk 2]
```diff
@@ -13,7 +14,7 @@ object Dependencies {
     }
 
     playJsonVersion match {
-      case jsonVersion if jsonVersion == playJson25 => Seq("com.typesafe.play" %% "play-json" % jsonVersion)
+      case jsonVersion if Seq(playJson24, playJson25).contains(jsonVersion) => Seq("com.typesafe.play" %% "play-json" % jsonVersion)
```
Comment (lolgab) [inline project/Dependencies.scala]:
```suggestion
      case `playJson24` | `playJson25` => Seq("com.typesafe.play" %% "play-json" % jsonVersion)
```

Comment (lolgab) [reply project/Dependencies.scala]: You can match directly on values

File: build.sbt (hunk 1 above)
Comment (machadoit) [reply build.sbt]: Changed there and the line bellow

File: project/Dependencies.scala (hunk 2 above)
Comment (machadoit) [reply project/Dependencies.scala]: cute

**Pull request merged**